src/styles/styles.js

```diff
@@ -742,6 +742,10 @@ const styles = {
         borderColor: themeColors.border,
     },
 
+    br0: {
+        borderRadius: 0,
+    },
+
```

Comment:
I'd move this to where we have `noRightBorderRadius` and `noLeftBorderRadius` declared and update the name to `noBorderRadius` for consistency